app/themes/semantic.js

```diff
@@ -0,0 +1,86 @@
+import Default from 'ember-models-table/themes/semanticui';
+
+
+export default Default.extend({
+  components: {
+    'pagination-simple'  : 'components/ui-table/simple-pagination',
+    numericPagination    : 'components/ui-table/numeric-pagination',
+    'table-footer'       : 'components/ui-table/table-footer',
```

Comment:
For all of this, doesn't something like `ui-table/table-footer` work ? (Without the `components/` part)

Reply:
I didn't check. Will do it now

Reply:
No doesn't work

Reply:
Okay cool! Thanks for the update @mayank8318 :)